gradescope_utils/autograder_utils/json_test_runner.py

```diff
@@ -174,6 +181,9 @@ def run(self, test):
             total_score += test["score"]
         self.json_data["score"] = total_score
 
+        if self.post_processor:
```

Comment:
nit: would be a bit more pythonic to do the following, but not a big deal either way.
```suggestion
        if self.post_processor is not None:
```

Reply:
My Python code always ends up looking like Ruby lol.

Reply:
I feel you. I find myself forgetting the `:`s and putting unnecessary `end`s every time.